### ISSUE TITLE
Can't connect to Dropbox

### ISSUE THREAD

**Issue by rammzomar** [cgollner/unclouded#130]
Hi. I can't connect to Dropbox which is the only reason I bought the product (duplicate deletion).

Please help.

If it helps, I have 2 factor security activated (sms).

**Comment by Fontalamh** [comment]
I can't sign in either. I don't have 2 factor security.

**Comment by MD999** [comment]
Likewise - I don't have 2 factor either. Very frustrating - that's the only reason I bought the app - full cloud integration!

**Comment by cgollner** [comment]
Hey guys, the Dropbox service that Unclouded uses has been shutdown. Working on a fix to use the new service. Sorry for the inconvenience!

**Comment by rammzomar** [comment]
Thanks Christian - look forward to it working again soon. Please let us
know an ETA once you think you have one.

**Comment by kddickey** [comment]
Any news on this? My Dropbox account is showing that I am connected to the app, but, it doesn't show up in the list of connected accounts.

**Comment by lucasfariaslf** [comment]
Hi @cgollner first of all: awesome app! Your last message was a month ago, any news on the fix?

**Comment by Jannto** [comment]
Yes, the same problem with Dropbox

**Comment by y0urS3LF85** [comment]
Same here...  Won't let me add my Dropbox Acount.  I get as far as to allowing the app permissions and that's it...

**Comment by StephenKoller** [comment]
Same. Purchased unlimited accounts upgrade feature just so I could add Dropbox along with local storage and Drive. Really disappointed...any word on when you'll have the new integration ready?

**Comment by asitemade4u** [comment]
Same here, just bought Unclouded and find frustrating not to be able to connect to my Dropbox (also without two factor auth).

**Comment by jboyd1111** [comment]
Any update on this? I have 2-factor authentication but am not able to link Dropbox account.

**Comment by rammzomar** [comment]
Hi Christian

We don't want to keep chasing you if there is NO PLAN to fix Unclouded for
Dropbox - which we all clearly LOVE as a product for Dropbox.

Can you please reply to our questions with a *realistic* timeline, or, if
you *DON'T* plan on fixing it, please let us know also so we can sadly move
on?

Many thanks in advance,

Rammiz

**Comment by asitemade4u** [comment]
I agree, this is the least you owe us.

Besides, we have bought your product and it was advertized as allowing DropBox as a cloud provider. In case you cannot figure out how to connect it, do you intend to reimburse us?


Stephen Sinclair ASiteMade4U On Mon, Nov 27, 2017 at 11:20pm, rammzomar < notifications@github.com [notifications@github.com] > wrote:
Hi Christian

We don't want to keep chasing you if there is NO PLAN to fix Unclouded for
Dropbox - which we all clearly LOVE as a product for Dropbox.

Can you please reply to our questions with a *realistic* timeline, or, if
you *DON'T* plan on fixing it, please let us know also so we can sadly move
on?

Many thanks in advance,

Rammiz

**Comment by makalove** [comment]
Same issue, same complaint. I paid for the app because of the claim of Dropbox integration. If you don't intend to be responsive to your users and maintain functionality, stop selling your app!

**Comment by lucasfariaslf** [comment]
I find it weird that there's no obvious/easy way to repport an app in
Google Play. Mnay people are still paying for this and we're not even sure
this the dev is alive.

**Comment by rammzomar** [comment]
I'm guessing it's time to do review the app on Google Play ... being
totally ignored.

**Comment by rikkiprince** [comment]
Give @cgollner a break! It's a free app, and it's just been the holiday period.

**Comment by asitemade4u** [comment]
No, not free, paid.


Stephen Sinclair ASiteMade4U On Sun, Jan 07, 2018 at 7:22pm, Rikki < notifications@github.com [notifications@github.com] > wrote:
Give @cgollner [https://github.com/cgollner] a break! It's a free app, and it's just been the holiday period.

**Comment by makalove** [comment]
Right, quite a few of us paid for the app because of the Dropbox integration. And @cgoliner hasn't responded in three months. It's nothing to do with holidays.

**Comment by rammzomar** [comment]
And it's been since the beginning of OCTOBER since I logged this but a few
weeks when the issue started. Not the Holiday Season

**Comment by burr1981** [comment]
Hi I already purchased the unlock for to add more than one account, I see there slot of complaints regarding this & no1 if replying, so instead of moaning right away if there any error log I can send you that will help, I'm also using honorary 6x on android 6.0 the Dropbox part used to work but they must of changed something in the back end to block it & I've noticed slot of people have started moaning on the place store, while 99% of the app works Dropbox is the main stream online storage for most so you need to try fix this or offer a refund or unlock maybe one of the other IAP so people stop moaning while a fix is found, but pulling a no man sky on us isn't helping. I'll be the biggest fan you'll ever lose, sincerely yours, Stan (quote eminem-stan)

**Comment by daniloaja** [comment]
My biggest interest was for the use of Dropbox, since there is no short-term solution I would like the refund of the amount paid

**Comment by rikkiprince** [comment]
@daniloaja I reckon that's something for you to take up with Google Play, rather than on the github issue tracker.

**Comment by rammzomar** [comment]
On google play it leads you here to log the issue (his design).

**Comment by chadyj** [comment]
+1

**Comment by demmeis** [comment]
+1 I also purchased the paid version specifically to add my Dropbox account and am frustrated by the fact that it isn't working.

**Comment by rammzomar** [comment]
1 star rate it on play. hes just ignoring us now.

**Comment by krawazi** [comment]
I have the same problem as all the users above and described it also in issue #140  - it seems that many  users have the same problems (and not even only since a few days) and no one is able to show or present a solution - thats bitter

**Comment by yayabiko** [comment]
I think the developer has stopped supporting the project.  Last update to the app was August 8, 2016.  The developer website associated with the app no longer even mentions Unclouded.  http://cgollner.x10.mx/?page_id=12
